src/main/java/edu/kis/powp/jobs2d/command/utils/JsonCommandReader.java

```diff
@@ -0,0 +1,22 @@
+package edu.kis.powp.jobs2d.command.utils;
+
+import com.google.gson.stream.JsonReader;
+
+import java.io.FileNotFoundException;
+import java.io.FileReader;
+import java.util.logging.Logger;
+
+public class JsonCommandReader {
+    private final static Logger logger = Logger.getLogger(Logger.GLOBAL_LOGGER_NAME);
+
+    public static JsonReader read(String path) {

```

Comment:
Too much classes. Such, a static method could be added somewhere else.

Reply:
Removed this class